pkg/api/scylla/v1alpha1/types_operatorconfig.go

```diff
@@ -25,6 +25,10 @@ type ScyllaOperatorConfigSpec struct {
 	// Setting this field renders your cluster unsupported. Use at your own risk.
 	// +optional
 	UnsupportedPrometheusVersionOverride *string `json:"unsupportedPrometheusVersionOverride,omitempty"`
+
+	// clusterDomainOverride allows to adjust Kubernetes cluster domain used by the operator as a cluster identifier.
+	// +optional
+	ClusterDomainOverride *string `json:"clusterDomainOverride,omitempty"`
```

Comment:
```suggestion
	ClusterDomain *string `json:"clusterDomain,omitempty"`
```
this should be valid to set, and if set it should disable the autodetection, so it's not actually an override

Reply:
done